src/Console/SelfUpdateCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace SilverStripe\Upgrader\Console;
+
+use SilverStripe\Upgrader\Util\UpdateChecker;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Command to automate the installation of the latest version of the upgrader.
+ */
+class SelfUpdateCommand extends Command
+{
+
+    protected function configure()
+    {
+        $this->setName('self-update')
+            ->setDescription('Get the latest version of the SilverStripe upgrader.')
+            ->setDefinition([
+                new InputOption('yes', 'y', InputOption::VALUE_NONE, 'Do not ask for confirmation.'),
+                new InputOption('rollback', 'r', InputOption::VALUE_NONE, 'Rollback the latest self-update.')
```

Comment:
Would it be worth adding specific version numbers at all? 

E.g: `upgrade-code self-update 1.4.0`

Reply:
I don't think the library I'm using for this support upgrading to a specific version. Just to the latest version available. In any case where trying to nudge people to always be on the latest version.